public/assets/images/icons.svg

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><symbol id="dollar" viewBox="0 0 511.613 511.613">
```

Comment:
a good practice is to minimize SVGs so that the files are not so large. This [tool](https://jakearchibald.github.io/svgomg/) is really useful and intuitive to use. I also use [ImageOptim](https://imageoptim.com/mac) for desktop.

Reply:
Thanks - yes!!!